webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/BreadcrumbSwitcherPopover.js

```diff
@@ -37,9 +38,9 @@ const BreadcrumbSwitcherPopover = ({
       </div>
     );
   } else {
-    const handleItemClick = (item) => {
-      onResourceClick(item);
-      if (item.onClick) item.onClick();
+    const handleItemClick = (e, item) => {
+      onClose(item);
+      if (onClickHandler) onClickHandler(e, item.url);
```

Comment:
I understand it now when you renamed it, thanks. However, I still think it's the same thing from this component's (`BreadcrumbSwitcherPopover`) point of view. It should IMHO be just `onClick(e, item)` here and `BreadcrumbSwitcher` should contain the logic on the lines above.